Automattic/wp-calypso
# Email stats open rate UI

#### Proposed Changes

This PR adds the open rate to the email stats. 

- It introduces the concept of period bound stats & all-time stats. Because of this I refactored the Redux actions and selectors a bit.
- It uses a component similar to `<HighlightCard />` to render the info on top.
- It also fetched the post, so we're able to show the title. While this is being fetched, a spinner is shown.

<img width="1176" alt="CleanShot 2023-01-10 at 14 51 58@2x" src="https://user-images.githubusercontent.com/528287/211569083-1595f64e-cd81-46a1-8228-c752f5c8efb6.png">

#### Testing Instructions



* Visit `/stats/email/open/<site>/day/<postid>?flags=newsletter/stats`. You can add the startDate parameter, to load the stats for a specific date.
* Open rate should be at the top

#### Pre-merge Checklist
